torch/_dynamo/variables/tensor.py

```diff
@@ -510,7 +480,9 @@ def create(cls, tx, proxy, sym_num, **options):
         proxy.node.meta["example_value"] = sym_num
 
         if isinstance(sym_num, (sympy.Integer, int)):
-            return ConstantVariable(int(sym_num))
+            if isinstance(sym_num, sympy.Integer):
```

Comment:
This is a preeexisting condition, but it doesn't seem to me that `sym_num` should ever be an sympy.Integer

Reply:
Agreed.